### ISSUE TITLE
Weeds work in composter

### ISSUE THREAD

**Issue by m0thknight** [AstralOrdana/Immersive-Weathering#241]
**Feature:**

my farm ends up with tons of the "weeds" item which i end up just throwing on the ground and waiting to despawn. It would be nice if they worked in composters or had a tangible use.

**Comment by nekomaster** [comment]
Yeah it'd be really nice if there was a use for weeds like composting them, like perhaps one might do IRL.

**Comment by RandomGuy91** [comment]
agreed

**Comment by SusuPoofyPants** [comment]
I've been been playing with this mod for a couple of weeks and been thinking this the whole time. Weeds should be composted :D

**Comment by AstralOrdana** [comment]
not sure why this isnt working, in code weeds are specified as compostable

**Comment by AstralOrdana** [comment]
please specify your versions and modloaders

**Comment by nekomaster** [comment]
> 

I'm using immersive_weathering-1.19.2-1.2.8-forge.jar in Minecraft 1.19.2 with Forge 43.2.4
Not only does it not show weeds as being compostable int he Usage window in JEI, you can't even use it on the compost bin.

For now I just used craft tweaker to add a "compost" recipe for weeds

**Comment by m0thknight** [comment]
Using fabric immersive weathering, same thing here. version immersive_weathering-1.19.2-1.2.8-fabric.jar, and its not appearing in JEI or able to put weeds into a composter at all.

**Comment by SusuPoofyPants** [comment]
im using the same as m0thknight, same thing

**Comment by itsdeadfrosty** [comment]
> Yeah it'd be really nice if there was a use for weeds like composting them, like perhaps one might do IRL.

you don't ever compost weeds. as they run the chance of growing in your garden. its too risky. your trash or burn em. instead, download the Seeds mod, the trash can in it gives your junk a use :)

**Comment by m0thknight** [comment]
> 



> > Yeah it'd be really nice if there was a use for weeds like composting them, like perhaps one might do IRL.
> 
> you don't ever compost weeds. as they run the chance of growing in your garden. its too risky. your trash or burn em. instead, download the Seeds mod, the trash can in it gives your junk a use :)

Seeds mod is only on forge and im playing fabric. also the mod should have a use for its items without having to download a separate mod.

**Comment by itsdeadfrosty** [comment]
Yeah and it should make you a sandwich and shine your shoes and teach you how to do you arithmetic too.

First of the mod doesn’t have to do jack crap. You just want it to have a function the exact way that you want. which is not how the world works. Stop being so entitled.

**Comment by mintynoura** [comment]
lol calm down. The developer already responded saying that weeds are supposed to be compostable.  I’m not sure why you used this bug report to advertise an entirely different mod, so honestly just chill out lol. This isn’t a comment section, so I’d rather you avoid responding to this comment. It’ll just clog the github with unnecessary snide remarks that I’m not going to bother replying to.

**Comment by itsdeadfrosty** [comment]
why would i care what you want? It's not an advertisement, it's a suggestion. And that's crazy, because it literally allows us to make comments. So it's almost like it's literally a comment section is the dumbest take you could have possibly picked. Now if you were to say it's not a recreational forum, maybe, sure, but I still don't care. If you can comment, I can comment and I don't have to obey your whims. You sound like a moron.

**Comment by SusuPoofyPants** [comment]
to further the point of this thread but also to enlighten anyone
interested, there *are* a few ways to safely compost weeds without wasting
them in the trash or a fire:) have a look online. There's always a risk of
them growing in ur garden regardless, so u might as well use them imo.
Depending on ur climate u can even fill a raised garden bed container with
weeds & plant old potatoes directly into them; I did that at my old place &
got great harvests from it. So weeds can be very useful & if u read the
whole thread you'd see the dev intended for them to be so in this mod too
but something seems to have gone wrong

anyway, i believe the idea is more immersion than realism, otherwise weeds
would still eventually pop up in mulch & rich soil, putting potatoes in
this type of composter would subsequently produce potato plants, composters
would otherwise produce compost instead of bone meal, etc.. ppl here are
just saying it would be a more immersive gardening experience if weeds were
a part of the activity beyond cluttering ur inventory. I'm not sure why
that bothers u so much but it's certainly not appropriate to respond with
insults & rudeness
